apps/cms/README.md

```diff
@@ -64,6 +64,24 @@ When you first spin up the app, you'll see only text rendered in place of images
 
 Then go back and refresh, and you should see images!
 
+## Configure Webhook for Scribe
+
+1. Navigate to the CMS and login
+1. Go to Settings > Webhooks
+1. Click on Create Webhook (the round plus button)
+1. Enter Scribe Webhook details and click save.
+
+| Field           | Value                                                                                                                           |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| Name            | `Scribe` (or any name you prefer)                                                                                               |
+| Method          | `POST`                                                                                                                          |
+| URL             | `SCRIBE_URL` + `/webhooks/directus`                                                                                             |
```

Comment:
Would this be `http://localhost:[scribe port]/webhooks/directus` locally?

Reply:
That is correct